siac/hostcmd.go

```diff
@@ -62,6 +62,14 @@ func hostconfigcmd(param, value string) {
 		p.Mul(p, big.NewRat(1e24/1e9, 4320))
 		value = new(big.Int).Div(p.Num(), p.Denom()).String()
 	}
+	// parse sizes of form 10GB, 10TB, 1TiB etc
+	if param == "totalstorage" || param == "minfilesize" || param == "maxfilesize" {
+		var err error // must be pre-declared because value is
```

Comment:
scoping in go can get annoying sometimes.

This PR is good enough, you're never really done polishing code and if you don't draw the line you'll only ever write 1 feature :P